### PR TITLE
Update Content

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Update content.js
* Update questions.js
* Some spec updates and refactors

### CODE REVIEW

File: spec/support/pages/definition_of_increased_risk_page.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Page Object that helps navigate through questions related
 # to defining the increased risk of kidney transplants
 class DefinitionOfIncreasedRiskPage < Page
-  def complete_lesson(pages_count = 7)
-    pages_count.times do
+  PAGES_IN_DEF_CHAP = 7
+
+  def complete_lesson(pages = PAGES_IN_DEF_CHAP)
+    pages.times do
       self.next
```
Comment (ericcf) [inline spec/support/pages/definition_of_increased_risk_page.rb]: Why is `self` necessary here? Isn't it implicit?

File: spec/informme_helper.rb [hunk 2]
```diff
@@ -15,3 +15,9 @@
 
 Dir[File.join(File.dirname(__FILE__), "/support/**/*.rb")]
   .each { |f| require f }
+
+RSpec.configure do |config|
+  config.before :all do
+    $stdout = File.open(File::NULL, "w")
```
Comment (ericcf) [inline spec/informme_helper.rb]: What's being written to stdout?

File: spec/support/pages/definition_of_increased_risk_page.rb (hunk 1 above)
Comment (michaelwehrley) [inline spec/support/pages/definition_of_increased_risk_page.rb]: This annoying log: "posting the following object to this URL: https://app2.cbits.northwestern.edu/prImporter"

File: spec/informme_helper.rb (hunk 2 above)
Comment (michaelwehrley) [inline spec/informme_helper.rb]: This annoying log: "posting the following object to this URL: https://app2.cbits.northwestern.edu/prImporter"

File: spec/support/pages/definition_of_increased_risk_page.rb (hunk 1 above)
Comment (michaelwehrley) [inline spec/support/pages/definition_of_increased_risk_page.rb]: ahhh, yes, but `next` is a reserved word and I can't just write it here.

**Pull request merged**